tests/org.pitest.pitclipse.ui.tests/src/org/pitest/pitclipse/ui/swtbot/PitNotifier.java

```diff
@@ -1,43 +0,0 @@
-/*******************************************************************************
```

Comment:
This file has been removed!

Reply:
Is now using an independent extension point with the class PitResultsNotifier.

Reply:
But the extension point is still there and it looks like the new mechanism is not working anyway... why do you need to introduce a new mechanism?

Reply:
Which extension point do you mean?
The fragment.xml only has the new extension point.

I don't see why the new mechanism is not working. What is not working?
I reused the mechanism which is used to supply the Pit Summary view directly and removed the other dependency because  this one would function as this:

1. pit started
2. pit done and notify pit summary view with the html files location.
3. notify that pit summary view has results
4. grab html text from browser of summary view and parse this
5. use parsed html text for proof of tests

New implementation of mine:
1. pit started
2. pit done notify summary view and PitResultsNotifier
3. Parse html file itself and present it for tests

So I deleted the dependency to the pit summary view, which is not necessary.

Reply:
> But the extension point is still there and it looks like the new mechanism is not working anyway... why do you need to introduce a new mechanism?

If you meant the class with the extension point "PitResultsView", I'm sorry I forgot to delete it. It is now gone in the newest commit, because it is not used.

Reply:
This class was crucial for synchronization if I remember correctly and it was also properly reset before running a new PIT run

Reply:
The synchronization is now done with PitResultsNotifier and works well as far as  the tests showed.
It is also reset in the same place as the old one. The behavior is almost the same as before, but without relying on the Pit Summary View.

Reply:
@LorenzoBettini can I assume this is resolved?